plugin/vcs/vcs.go

```diff
@@ -117,6 +125,12 @@ type Provider interface {
 	// repositoryID: the repository ID from the external VCS system (note this is NOT the ID of Bytebase's own repository resource)
 	FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx common.OauthContext, instanceURL string, repositoryID string) ([]*RepositoryMember, error)
 
+	// Fetch all repository list within the scope of the given user
```

Comment:
We should be explicit that we are fetching repositories in which the user has a maintainer role here.